PdfSharpCore/Utils/FontResolver.cs

```diff
@@ -58,23 +58,37 @@ static FontResolver()
             throw new NotImplementedException("FontResolver not implemented for this platform (PdfSharpCore.Utils.FontResolver.cs).");
         }
 
-        static string[] resolveLinuxFontFiles() {
-            List<string> fonts=new List<string>();
-            Regex confRegex=new Regex("<dir>(?<dir>.*)</dir>",RegexOptions.Compiled);
-            Regex ttfRegex=new Regex(@"\.ttf", RegexOptions.Compiled | RegexOptions.IgnoreCase);
-            using (StreamReader reader=new StreamReader(File.OpenRead("/etc/fonts/fonts.conf"))) {
-                string line=null;
-                while((line=reader.ReadLine())!=null) {
-                    Match m=confRegex.Match(line);
-                    if (!m.Success) continue;
-                    string val=m.Groups["dir"].Value.Replace("~",System.Environment.GetEnvironmentVariable("HOME"));
-                    if (!Directory.Exists(val)) continue;
-                    foreach(string dir in Directory.EnumerateDirectories(val)) 
-                        foreach(string s in Directory.EnumerateFiles(dir).Where(x=>ttfRegex.IsMatch(x))) 
-                            fonts.Add(s);
+        static string[] ResolveLinuxFontFiles()
+        {
+            var fontList = new List<string>();
+            var confRegex = new Regex("<dir>(?<dir>.*)</dir>", RegexOptions.Compiled);
+            var ttfRegex = new Regex(@"\.ttf", RegexOptions.IgnoreCase | RegexOptions.Compiled);
+
+            using (var reader = new StreamReader(File.OpenRead("/etc/fonts/fonts.conf")))
+            {
+                string line;
+                while ((line = reader.ReadLine()) != null)
+                {
+                    var match = confRegex.Match(line);
+                    if (!match.Success) continue;
+
+                    var path = match.Groups["dir"].Value.Replace("~", Environment.GetEnvironmentVariable("HOME"));
+                    if (!Directory.Exists(path)) continue;
+
+                    foreach (var enumerateDirectory in Directory.EnumerateDirectories(path))
+                    {
+                        var pathFont = Path.Combine(path, enumerateDirectory);
+
+                        foreach (var strDir in Directory.EnumerateDirectories(pathFont))
```

Comment:
Is this by observation or is it a known convention to have font files two levels deep? Is this universal for all fonts folders or does this only apply to `/usr/share/fonts`? Think it would just be simpler and more robust to just use [`Directory.EnumerateFiles(path, "*.ttf", SearchOption.AllDirectories)`](https://docs.microsoft.com/en-us/dotnet/api/system.io.directory.enumeratefiles?view=net-5.0#System_IO_Directory_EnumerateFiles_System_String_System_String_System_IO_SearchOption_) (and also skip the regex match and use the built in wildcard support instead).